devtools/docker-compose.yml

```diff
@@ -0,0 +1,23 @@
+version: '3.4'
+
+services:
+  eventgridemulator:
+    image: eventgridemulator
+    build:
+      context: ../src
+      dockerfile: ../src/EventGridEmulator/Dockerfile
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
```

Comment:
We should not recommend to specify ASPNETCORE_ENVIRONMENT and it will fallback to production environment. This is done in the README as well